commands/scan/buildscan.go

```diff
@@ -183,7 +183,13 @@ func (bsc *BuildScanCommand) runBuildScanAndPrintResults(xrayManager *xray.XrayS
 			}
 		}
 	}
-	err = utils.RecordSecurityCommandOutput(utils.ScanCommandSummaryResult{Results: scanResults.GetSummary(), Section: utils.Build})
+	err = utils.RecordSecurityCommandSummary(utils.NewBuildScanSummary(
+		scanResults,
+		bsc.serverDetails,
+		bsc.includeVulnerabilities,
+		bsc.buildConfiguration.GetProject() != "",
```

Comment:
couldn't we get violations from watches? we assume only Project in this case...

Reply:
no watches flag for build scan, only project can be provided as violation context in this cmd
![image](https://github.com/user-attachments/assets/26856328-95bd-4c06-a138-9e33610d92ad)